src/promptflow/tests/sdk_cli_test/e2etests/test_cli.py

```diff
@@ -67,17 +67,24 @@ def test_pf_version(self, capfd):
         out, _ = capfd.readouterr()
         assert "0.0.1\n" in out
 
-    def test_basic_flow_run(self, capfd) -> None:
+    def test_create_file_data_flow_new_relative_path(self, capfd) -> None:
+        new_cwd = Path(os.getcwd()).parent.parent
+        NEW_RUNS_DIR = "./src/promptflow/tests/test_configs/runs"
+        NEW_FLOWS_DIR = "./src/promptflow/tests/test_configs/flows"
+        NEW_DATA_DIR = "./src/promptflow/tests/test_configs/datas"
         # fetch std out
         run_pf_command(
             "run",
             "create",
-            "--flow",
-            f"{FLOWS_DIR}/web_classification",
+            "--file",
+            f"{NEW_RUNS_DIR}/sample_bulk_run.yaml",
             "--data",
-            f"{DATAS_DIR}/webClassification3.jsonl",
+            f"{NEW_DATA_DIR}/webClassification3.jsonl",
             "--name",
             str(uuid.uuid4()),
+            "--flow",
```

Comment:
Maybe we need to discuss about the behavior here. I think we are only replacing YAML contents in param overrides. So the path provided here should be relative to the YAML file